### DISCUSSION TITLE
How do you setup a custom local LLM that is compatible with OpenAI?

### DISCUSSION THREAD

**Discussion by FrozzDay** [zed-industries/zed#11785] [Feature Requests]
So, there is lot of public model we can try, and i see some people already using ollama with Zed. But i dont see any clear guide how to do that.

**Comment by tidely** [comment]
There was just a patch sent out that provides documentation for setting up OpenAI API compatible endpoints: [Commit](https://github.com/zed-industries/zed/commit/4766b41e96d325014aaddc9855b744948341eb70)
